ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master_metal.yaml

```diff
@@ -0,0 +1,58 @@
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/ocp/ubi-python-311:9
+      USER 0
+releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.18"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: e2e-rds-core-single-node
```

Comment:
What do you mean by single node?
This is a MNO deployment right?

Reply:
Thanks for this catch. Updating to mno.

Reply:
MNO is already the default, I was curious why you call it single-node